stdlib/InteractiveUtils/src/InteractiveUtils.jl

```diff
@@ -138,10 +138,18 @@ function versioninfo(io::IO=stdout; verbose::Bool=false)
     println(io, "  LIBM: ",Base.libm_name)
     println(io, "  LLVM: libLLVM-",Base.libllvm_version," (", Sys.JIT, ", ", Sys.CPU_NAME, ")")
 
-    env_strs = [String[ "  $(k) = $(v)" for (k,v) in ENV if occursin(r"JULIA", k)];
-                (verbose ?
-                 String[ "  $(k) = $(v)" for (k,v) in ENV if occursin(r"PATH|FLAG|^TERM$|HOME", k)] :
-                 [])]
+    function is_nonverbose_env(k::String)
+        return occursin(r"JULIA|^LD_LIBRARY_PATH$|^DYLD_LIBRARY_PATH$", k)
```

Comment:
```suggestion
        return occursin(r"^JULIA_|^DYLD_|^LD_", k)
```

Perhaps worth always printing all of these? We don't normally include the PATH variables, since users might consider it to be secret in rare cases, and it also just generally is not that interesting.

Reply:
Yeah I'd rather not print `PATH` unless `verbose=true`, because `PATH` is often very long.